### ISSUE TITLE
question for the config in SparseSelfAttention

### ISSUE THREAD

**Issue by Ultra11nstinct** [microsoft/DeepSpeed#1993]
Hi, 
I have some questions for the use the 'key_padding_mask'.
In torch.nn.multiheadattention, the mask for keys to join local attention is set to 'False' for 'key_padding_mask', and 'True' for the keys that will not join the calculation of local attention.
If I want to avoid some attend keys in local attention, should I set 0 or 1 in the corresponding position in 'key_padding_mask' in SparseSelfAttention?

Looking forward to your reply.
